src/peer_digest.cc

```diff
@@ -754,25 +729,11 @@ peerDigestPDFinish(DigestFetchState * fetch, int pcb_valid, int err)
         else
             debugs(72, 2, "received valid digest from " << host);
     }
-
-    cbdataReferenceDone(fetch->pd);
 }
 
-/* free fetch state structures
- * must be called only when fetch cbdata is valid */
 static void
 peerDigestFetchFinish(DigestFetchState * fetch, int /* err */)
 {
-    assert(fetch->entry && fetch->request);
-
-    if (fetch->old_entry) {
-        debugs(72, 3, "peerDigestFetchFinish: deleting old entry");
-        storeUnregister(fetch->old_sc, fetch->old_entry, fetch);
-        fetch->old_entry->releaseRequest();
-        fetch->old_entry->unlock("peerDigestFetchFinish old");
-        fetch->old_entry = nullptr;
-    }
-
     /* update global stats */
     statCounter.cd.kbytes_sent += fetch->sent.bytes;
```

Comment:
We are not going to fix this in this PR, but just for my information, is this "kilobytes += bytes" math buggy?